cura_app.py

```diff
@@ -3,8 +3,23 @@
 # Copyright (c) 2015 Ultimaker B.V.
 # Cura is released under the terms of the AGPLv3 or higher.
 
+import os
 import sys
 
+#WORKAROUND: GITHUB-704 GITHUB-708
+# It looks like setuptools creates a .pth file in
+# the default /usr/lib which causes the default site-packages
+# to be inserted into sys.path before PYTHONPATH.
+# This can cause issues such as having libsip loaded from
+# the system instead of the one provided with Cura, which causes
+# incompatibility issues with libArcus
+if "PYTHONPATH" in os.environ.keys():                # If PYTHONPATH is used
+    PYTHONPATH_real = os.path.realpath(os.environ["PYTHONPATH"])
```

Comment:
That looks interesting.. It works only on the first element, but it also makes sense - it just cares about the beginning of the string.

```
>>> os.path.realpath("BLA:/foo/bar:foobar")
'/home/thopiekar/BLA:/foo/bar:foobar'
```

That brings me to a good idea...